lib/src/sql/function.rs

```diff
@@ -347,6 +347,7 @@ fn function_math(i: &str) -> IResult<&str, &str> {
 			tag("math::ceil"),
 			tag("math::fixed"),
 			tag("math::floor"),
+			tag("math::pow"),
```

Comment:
Can we keep these in alphabetical order?

Reply:
... Line 364

Reply:
Is there some kind of logic to the groupings?